portfolio/src/main/webapp/script.js

```diff
@@ -46,25 +46,19 @@ function grow() {
   };
   const image = document.getElementById('basil');
   console.log(`Let's grow some basil`);
-
-  function updateImagePerSec(imgIndex) {
-    setTimeout(function () {
-      console.log('Some time has passed ...');
-      // Change the image
-      const imgSrc = `images/basil${imgIndex}.jpg`;
-      image.src = imgSrc;
-      image.alt = srcToAlt[imgSrc];
-      image.width = '400';
-      image.height = '400';
-      imgIndex++;
-      if (imgIndex <= 6) {
-        updateImagePerSec(imgIndex); // Keep growing
-      } else {
-        console.log(`This's the latest photo I have! The basil will keep growing ...`);
-      }
-    }, 1000); // Update the image every 1000 ms = 1 sec
+  
+  for (let imgIndex = 1; imgIndex <= 6; imgIndex++) {
+    updateImg(imgIndex);
   }
 
-  // Start with imgIndex=1
-  someTimeLater(1);
+  function updateImg(imgIndex) {
+    setTimeout(
+      function () {
+        const imgSrc = `images/basil${imgIndex}.jpg`;
+        image.src = imgSrc;
+        image.alt = srcToAlt[imgSrc];
+        image.width = '400';
+        image.height = '400';
+      }, 1000 * imgIndex);
```

Comment:
Haha nice work around! Though idk if its more readable than what i suggested. Its because its not clear why you are computing these unique looking time.

Reply:
Tbh your solution and mine both have pros and cons. You are fine to do whichever just have a comment explaining what is happening.

Reply:
Hey sorry what was the solution you are referring to? I can't seem to find it. Curious to see what that looks like!

Reply:
Nvm I found it haha sorry about the confusion.